website/docs/tutorials/07-testing/02-cheatcodes/reject.md

```diff
@@ -0,0 +1,30 @@
+# `reject`
+```python
+def reject() -> None:
+```
+Marks example as bad, thus preventing fuzz tests from using it as a falsifying example.
+
+`reject()` is equivalent to `assume(False)`.
+
+```cairo
+%lang starknet
+
+@external
+func test_function_that_takes_nonzero_argument{syscall_ptr : felt*, range_check_ptr}(value):
+    if value != 0:
+        %{ reject() %}
+        assert 0 = 0
+    end
+
+    # ...
+
+    return ()
+end
+```
+:::warning
+This cheatcode is only available in [fuzz tests](../fuzzing).
+:::
```

Comment:
it's definitely more important fragment than the example and the statement that reject is equivalent to assume. I would put this box after the `Marks example as bad` statement